daonode/src/main/java/bisq/daoNode/DaoNode.java

```diff
@@ -15,7 +15,7 @@
  * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
  */
 
-package bisq.daonode;
+package bisq.daoNode;
```

Comment:
Codacy caught this:  Package name contains upper case characters.

Reply:
Oh damn codacy... In Bisq 2 we decided to use camelCase in package names (even its not official java convention) as its better readable with longer package names. But I will change it here to not get in long painful discussion with the Codacy dictatorship ;-)